lib/pure/strtabs.nim

```diff
@@ -141,10 +141,8 @@ template get(t: StringTableRef, key: string) =
   var index = rawGet(t, key)
   if index >= 0: result = t.data[index].val
   else:
-    when compiles($key):
-      raise newException(KeyError, "key not found: " & $key)
-    else:
-      raise newException(KeyError, "key not found")
+    raise newException(KeyError, "key not found: " & key)
```

Comment:
Won't this fail to compile if `key` doesn't have a `$` method?

Reply:
But key is string, it doesn't need $?

Reply:
I think @Varriount meant to write this comment under the other change - in `critbits.nim`, where `key` might not be a string?